### PR TITLE
negar/virtualws

### PR DESCRIPTION
**Pull request by negar-binary**
create new form and page for virtual account creation through ws

### CODE REVIEW

File: src/sass/static/virtual_acc_opening.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#virtual-form{
+	text-align: center;
```
Comment (jy-binary) [inline src/sass/static/virtual_acc_opening.scss]: why so much custom CSS? Please try to use only our standard styling from https://www.binary.com/styles

File: src/templates/haml/home2/styles.html.haml [hunk 2]
```diff
@@ -0,0 +1,519 @@
+- stash(
```
Comment (jy-binary) [inline src/templates/haml/home2/styles.html.haml]: why is this file duplicated?

File: src/templates/haml/home2/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,120 @@
+- stash(
```
Comment (jy-binary) [inline src/templates/haml/home2/index.html.haml]: I don't think you need a home2. Home2 is just for a/b testing. This project isn't about a/b testing.

File: src/javascript/binary/static/verify_email.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+//get gtm layer from ashkan before release
```
Comment (raunakkathuria) [inline src/javascript/binary/static/verify_email.js]: remove this

Comment (ashkanx) [inline src/javascript/binary/static/verify_email.js]: Let my name be there :)

Comment (borisyankov) [inline src/javascript/binary/static/verify_email.js]: you can add // todo:
to comments like this, and then grep them before committing ;)

Comment (negar-binary) [inline src/javascript/binary/static/verify_email.js]: @borisyankov good idea, thanks :D

**Pull request merged**